src/server/ua_discovery_mdns.c

```diff
@@ -547,7 +549,7 @@ mdns_find_record(mdns_daemon_t *mdnsDaemon, unsigned short type,
 
 /* set record in the given interface */
 static void
-mdns_set_address_record_if(UA_DiscoveryManager *dm, const char *fullServiceDomain,
+mdns_set_address_record_if(const char *fullServiceDomain,
                            const char *localDomain, char *addr, UA_UInt16 addr_len) {
```

Comment:
Style: The parameters could be distributed differently over two lines. Now it looks strange.
```suggestion
mdns_set_address_record_if(const char *fullServiceDomain, const char *localDomain,
                           char *addr, UA_UInt16 addr_len) {
```